draft-privacypass-rate-limit-tokens.md

```diff
@@ -424,13 +424,14 @@ Its ABNF is:
     Sec-Token-Request-Blind = sf-binary
 ~~~
 
-The "Sec-Token-Count" is an Item Structured Header {{!RFC8941}}. Its
-value MUST be an Integer. This header is sent on Attester-to-Issuer
-requests ({{request-two}}), and contains the number of times a
-Client has previously received a token for an Origin. Its ABNF is:
+The "Sec-Token-Limit" is an Item Structured Header {{!RFC8941}}. Its
+value MUST be an Integer. This header is sent on Issuer-to-Attester
+responses ({{response-one}}), and contains the number of times a
+Client can retrieve a token for the requested Origin, set by the
```

Comment:
can retrieve a token for the requested Origin _within a policy window_, as set ..